delira/training/callbacks/logging_callback.py

```diff
@@ -0,0 +1,85 @@
+from delira.training.callbacks.abstract_callback import AbstractCallback
+from delira.logging import make_logger, BaseBackend
+import logging
+
+
+class DefaultLoggingCallback(AbstractCallback):
+    """
+    A default Logging backend which logs only the metrics; Should be
+    subclassed for additional logging
+    """
+
+    def __init__(self, backend: BaseBackend, max_queue_size: int = None,
```

Comment:
Can we define `at_epoch_end` and log the reduced (typically average) values of the metrics? Or is this done somewhere else?

Reply:
This should not be necessary. It should be done on a per-iter basis combined with the corresponding frequency

Reply:
Currently the reduced values are logged and I liked the feature because it is very easy to see which epoch is the best.

Reply:
So you want to log them in Addition to the ones with the frequency? We can certainly do this. What prefix/infix/suffix do you propose to distinguish them?

Reply:
Yes, I want them in addition to the ones with frequency 👍 I like `reduced_` as a prefix because the exact `reduce_fn` can vary.

Reply:
I would prefer `reduced_REDUCTIONTYPE_` and use the string passed to the keyword mapping as REDUCTIONTYPE

Reply:
This is not as easy, as I thought, since we would have to bypass the logging frequency for these values, which is a thing, that is not implemented inside the logger, since it is undesired behavior. Otherwise, we would only log the reduced values every `n` epochs (if  `n` is the given logging frequency)

Reply:
So @mibaumgartner and I discussed elsewhere and decided to maybe include this later, but not in this PR